MapboxDirections/MBRouteStep.swift

```diff
@@ -421,6 +421,35 @@ public enum ManeuverDirection: Int, CustomStringConvertible {
     }
 }
 
+@objc(MBRoadSide)
+public enum RoadSide: Int, CustomStringConvertible {
+    case left
+    case right
```

Comment:
Although this is pretty self explanatory, can you document this class it's properties?